---
title: Genomics and prevalence of bacterial and archaeal isolates from biogas-producing
  microbiomes
authors:
- Irena Maus
- Andreas Bremges
- Yvonne Stolze
- Sarah Hahnke
- Katharina G Cibis
- Daniela E Koeck
- Yong S Kim
- Jana Kreubel
- Julia Hassa
- Daniel Wibberg
- Aaron Weimann
- Sandra Off
- Robbin Stantscheff
- Vladimir V Zverlov
- Wolfgang H Schwarz
- Helmut König
- Wolfgang Liebl
- Paul Scherer
- Alice C McHardy
- Alexander Sczyrba
- Michael Klocke
- Alfred Pühler
- Andreas Schlüter
date: '2017-11-01'
publishDate: '2024-06-30T12:48:55.644579Z'
publication_types:
- article-journal
publication: '*Biotechnol. Biofuels*'
abstract: 'Background: To elucidate biogas microbial communities and processes, the
  application of high-throughput DNA analysis approaches is becoming increasingly
  important. Unfortunately, generated data can only partialy be interpreted rudimentary
  since databases lack reference sequences. Results: Novel cellulolytic, hydrolytic,
  and acidogenic/acetogenic Bacteria as well as methanogenic Archaea originating from
  different anaerobic digestion communities were analyzed on the genomic level to
  assess their role in biomass decomposition and biogas production. Some of the analyzed
  bacterial strains were recently described as new species and even genera, namely
  Herbinix hemicellulosilytica T3/55T, Herbinix luporum SD1DT, Clostridium bornimense
  M2/40T, Proteiniphilum saccharofermentans M3/6T, Fermentimonas caenicola ING2-E5BT,
  and Petrimonas mucosa ING2-E5AT. High-throughput genome sequencing of 22 anaerobic
  digestion isolates enabled functional genome interpretation, metabolic reconstruction,
  and prediction of microbial traits regarding their abilities to utilize complex
  bio-polymers and to perform specific fermentation pathways. To determine the prevalence
  of the isolates included in this study in different biogas systems, corresponding
  metagenome fragment mappings were done. Methanoculleus bourgensis was found to be
  abundant in three mesophilic biogas plants studied and slightly less abundant in
  a thermophilic biogas plant, whereas Defluviitoga tunisiensis was only prominent
  in the thermophilic system. Moreover, several of the analyzed species were clearly
  detectable in the mesophilic biogas plants, but appeared to be only moderately abundant.
  Among the species for which genome sequence information was publicly available prior
  to this study, only the species Amphibacillus xylanus, Clostridium clariflavum,
  and Lactobacillus acidophilus are of importance for the biogas microbiomes analyzed,
  but did not reach the level of abundance as determined for M. bourgensis and D.
  tunisiensis. Conclusions: Isolation of key anaerobic digestion microorganisms and
  their functional interpretation was achieved by application of elaborated cultivation
  techniques and subsequent genome analyses. New isolates and their genome information
  extend the repository covering anaerobic digestion community members.'
tags:
- Anaerobic digestion; Biomethanation; Defluviitoga tunisiensis; Fragment recruitment;
  Genome sequencing; Methanoculleus bourgensis;my papers
---
