---
title: Inference of phenotype-defining functional modules of protein families for
  microbial plant biomass degraders
authors:
- Sebastian Ga Konietzny
- Phillip B Pope
- Aaron Weimann
- Alice C McHardy
date: '2014-09-01'
publishDate: '2024-06-30T12:48:55.667571Z'
publication_types:
- article-journal
publication: '*Biotechnol. Biofuels*'
abstract: "BACKGROUND: Efficient industrial processes for converting plant lignocellulosic
  materials into biofuels are a key to global efforts to come up with alternative
  energy sources to fossil fuels. Novel cellulolytic enzymes have been discovered
  in microbial genomes and metagenomes of microbial communities. However, the identification
  of relevant genes without known homologs, and the elucidation of the lignocellulolytic
  pathways and protein complexes for different microorganisms remain challenging.
  RESULTS: We describe a new computational method for the targeted discovery of functional
  modules of plant biomass-degrading protein families, based on their co-occurrence
  patterns across genomes and metagenome datasets, and the strength of association
  of these modules with the genomes of known degraders. From approximately 6.4 million
  family annotations for 2,884 microbial genomes, and 332 taxonomic bins from 18 metagenomes,
  we identified 5 functional modules that are distinctive for plant biomass degraders,
  which we term ``plant biomass degradation modules'' (PDMs). These modules incorporate
  protein families involved in the degradation of cellulose, hemicelluloses, and pectins,
  structural components of the cellulosome, and additional families with potential
  functions in plant biomass degradation. The PDMs were linked to 81 gene clusters
  in genomes of known lignocellulose degraders, including previously described clusters
  of lignocellulolytic genes. On average, 70% of the families of each PDM were found
  to map to gene clusters in known degraders, which served as an additional confirmation
  of their functional relationships. The presence of a PDM in a genome or taxonomic
  metagenome bin furthermore allowed us to accurately predict the ability of any particular
  organism to degrade plant biomass. For 15 draft genomes of a cow rumen metagenome,
  we used cross-referencing to confirmed cellulolytic enzymes to validate that the
  PDMs identified plant biomass degraders within a complex microbial community. CONCLUSIONS:
  Functional modules of protein families that are involved in different aspects of
  plant cell wall degradation can be inferred from co-occurrence patterns across (meta-)genomes
  with a probabilistic topic model. PDMs represent a new resource of protein families
  and candidate genes implicated in microbial plant biomass degradation. They can
  also be used to predict the plant biomass degradation ability for a genome or taxonomic
  bin. The method is also suitable for characterizing other microbial phenotypes."
tags:
- (Ligno)cellulose degradation; Feature ranking; Gene clusters; LDA; Latent Dirichlet
  allocation; Pectin degradation; Phenotype-based identification of functional modules;
  Plant biomass degradation; Polysaccharide utilization loci; Probabilistic topic
  models;my papers
---
